cli/js/files.ts

```diff
@@ -20,25 +20,73 @@ import {
 /** Open a file and return an instance of the `File` object
  *  synchronously.
  *
- *       const file = Deno.openSync("/foo/bar.txt");
+ *       const file = Deno.openSync("/foo/bar.txt", { read: true });
```

Comment:
read is default right? It would be good to show a read example without the capability argument. Then another more complex example.

Reply:
Yes, It's default. Yeah, and I would mention about default value in docs